src/main/java/persistence/sql/ddl/ColumnQueryBuilder.java

```diff
@@ -0,0 +1,39 @@
+package persistence.sql.ddl;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ColumnQueryBuilder {
+    private static ColumnQueryBuilder queryBuilderDDL = new ColumnQueryBuilder();
+    private ColumnQueryBuilder() { }
+    public static ColumnQueryBuilder getInstance() {
+        return queryBuilderDDL;
+    }
+    public List<ColumnInfo> extract(Class<?> clazz) {
+        return Arrays.stream(clazz.getDeclaredFields()).map(ColumnInfo::extract)
+                .filter(ColumnInfo::isNotTransient).collect(Collectors.toList());
+    }
+
+    public String extractPrimaryKeyQuery(Class<?> clazz) {
+        List<ColumnInfo> columnInfos = extract(clazz);
+        StringBuilder sb = new StringBuilder();
+        List<ColumnInfo> primaryKey = columnInfos.stream().filter(ColumnInfo::isPrimaryKey).collect(Collectors.toList());
+        if(primaryKey.isEmpty()) {
+            throw new IllegalArgumentException("Entity에 Id로 정의된 column이 존재하지 않습니다.");
+        }
```

Comment:
위의 피드백이 반영된다면 이런 느낌의 코드가 가능하지 않을까요?
```suggestion
        List<ColumnInfo> primaryKey = columnInfos.getPrimaryKeys();
```